crates/runc-shim/build.rs

```diff
@@ -0,0 +1,23 @@
+use std::{process::Command, str::from_utf8};
```

Comment:
Need Copyright declaration in the first line. @mxpv Did we miss some file header check in CI?

Reply:
Done.